CHANGELOG.md

```diff
@@ -30,6 +30,150 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v9.1.0](https://github.com/shopsys/shopsys/compare/v9.0.4...v9.1.0) - 2020-12-21
+
+### [shopsys/framework]
+
+#### Added
+
+- [#1951] new phing target for changing environment
+
+#### Changed
+
+- [#1933] Optimization of calling replacing extended entities in queries, Thanks to [@pk16011990]
+- [#1994] added link into breadcrumb item in admin if uri is not null
+- [#2031] deprecated notice is triggered when deprecated method is called
+- [#2070] Do not use private in constraints, Thanks to [@pk16011990]
+- [#2079] throwable is now available in logger context when a cron fails on error
+- [#2080] optimization: flush the whole identity map when an order is created
+- [#2084] do not run checks for cart modifications on cart delete
+- [#2118] ConstantVisibilityRequiredSniff and ForceLateStaticBindingForProtectedConstantsSniff are ignored for sources folders only in project-base
+- [#2133] Enabled caching for DomainRouter and LocalizedRouter, Thanks to [@pk16011990]
+- [#2148] use DBAL native queries for DML instead of ORM Native SQL
+- [#2153] refactoring of NormalizeUrlTrailingSlashSubscriber
+
+#### Fixed
+
+- [#1760] added array_key_exists condition into to NotNullableColumnsFinder method
+- [#1901] fixed AdministratorFacade::changePassword() method
+- [#1952] JS validation: fix creating of prototype of compound item in collection, Thanks to [@pk16011990]
+- [#2064] fixed currencies menu text
+- [#2083] fixed checking of empty settings values
+- [#2091] fixed rendering currency symbol in admin grid
+- [#2095] Wrong annotation of return type in getFullPathsIndexedByIdsForDomain, Thanks to [@CarabineSK]
+- [#2142] fixed copying images when creating variants
+
+#### Removed
+
+- [#2020] removed SliderFormType excessive ID constraint, Thanks to [@milanStano]
+
+### [shopsys/frontend-api]
+
+#### Added
+
+- [#1891] added support for changing personal data and password to frontend API
+- [#1960] Added resolvers for orders
+- [#1996] added articles resolver
+- [#2002] Added special article resolvers
+- [#2047] add brands to frontend API
+- [#2057] added missing data to Product frontend API type
+- [#2066] added promoted products
+- [#2067] Added seo information to FEAPI
+- [#2068] added resolver for adverts into FEAPI
+- [#2100] added support for customer user registration to Frontend API
+- [#2110] FE API ordering products
+- [#2119] add support for subscribing for e-mail newsletter to FE API
+- [#2150] Added support to find entity by slug into FEAPI
+- [#2163] added search for products and categories in frontend API
+
+#### Changed
```

Comment:
can you please unify first characters (preferably lowercased) in the whole version (with exceptions for JS, FE API and class names)

Reply:
fixed

Reply:
unified